src/public/modules/forms/helpers/CsvMergedHeadersGenerator.ts

```diff
@@ -1,21 +1,32 @@
-const moment = require('moment-timezone')
-const keyBy = require('lodash/keyBy')
-const { CsvGenerator } = require('./CsvGenerator')
-const { getResponseInstance } = require('./response-factory')
-
-/**
- * @typedef {{
- *    _id: string,
- *    question: string,
- *    answer?: string,
- *    answerArray?: string[],
- *    fieldType: string,
- *    isHeader?: boolean,
- * }} DisplayedResponse
- */
-
-class CsvMergedHeadersGenerator extends CsvGenerator {
-  constructor(expectedNumberOfRecords, numOfMetaDataRows) {
+import { keyBy } from 'lodash'
```

Comment:
maintain the import of the specific library 'lodash/keyBy' so we minimise the bundle size of the frontend